EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/GeneratePanel.java

```diff
@@ -141,8 +145,11 @@ protected void populateItem(ListItem item) {
                     item.add(new Label("attributeInputDescription", getString("description.Undefined")));
                 } else {
                     item.add(new Label("attributeType", attribute.getType().getName()));
-                    item.add(new Label("attributeInputDescription", new StringResourceModel("description.${type}", this, new Model<TypeTreeNode>(attribute))));
+                    StringResourceModel inputDescriptionModel = new StringResourceModel("description.${type}", this, new Model<TypeTreeNode>
+                            (attribute));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') '(' should be on the previous line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=checkstyle%3AMethod_Param_Pad_BP2016)